api/requests/serializers.py

```diff
@@ -89,6 +89,40 @@ def get_target_url(self, obj):
             },
         )
 
+class RegistrationRequestCreateSerializer(RegistrationRequestSerializer):
```

Comment:
I don't get the point of creating requests via the serializer anymore, wasn't this all supposed to be done via the transitions?

Reply:
This needs to be removed. Artifact of old work. Again. I'm a dummy.